README.md

```diff
@@ -0,0 +1,63 @@
+# go-linq [![GoDoc](https://godoc.org/gopkg.in/ahmetalpbalkan/go-linq.v2?status.svg)](https://godoc.org/gopkg.in/ahmetalpbalkan/go-linq.v2) [![Build Status](https://travis-ci.org/ahmetalpbalkan/go-linq.svg?branch=v2.0)](https://travis-ci.org/ahmetalpbalkan/go-linq) [![Coverage Status](https://coveralls.io/repos/github/ahmetalpbalkan/go-linq/badge.svg?branch=v2.0)](https://coveralls.io/github/ahmetalpbalkan/go-linq?branch=v2.0)
+A powerful language integrated query (LINQ) library for Go.
+* Safe for concurrent use
+* Complete lazy evaluation
+* Supports arrays, slices, maps, strings, channels and
+custom collections (collection needs to implement Iterable interface
+and element - Comparable interface)
+* Parallel LINQ (PLINQ) *(comming soon)*
+
+## Installation
+
+    $ go get gopkg.in/ahmetalpbalkan/go-linq.v2
```

Comment:
Let's just use github here as well. I'm in the camp of people who are against pulling from gopkg.in as well. It's more likely that github will last longer :)